src/entity/users.rs

```diff
@@ -1,5 +1,7 @@
-use sea_orm::entity::prelude::*;
-use serde::Deserialize;
+use sea_orm::{entity::prelude::*, Set};
+use serde::{Deserialize};
+use rand::distributions::{Alphanumeric, DistString};
```

Comment:
Порядок импортов соблюсти
```suggestion
use rand::distributions::{Alphanumeric, DistString};
use sea_orm::{entity::prelude::*, Set};
use serde::{Deserialize};
```